components/lib/calendar/calendar.d.ts

```diff
@@ -269,19 +269,11 @@ export interface CalendarState {
      */
     overlayVisible: boolean;
     /**
-     * Current viewDate state as a string.
+     * Current viewDate state.
      */
-    viewDate: any;
+    viewDate: Nullable<Date>;
 }
 
-/**
- * Custom change event.
- * @see {@link CalendarProps.onChange}
- * @extends {FormEvent}
- * @event
- */
-interface CalendarChangeEvent extends FormEvent<Date | Date[] | string> {}
```

Comment:
Was string an improper type in this component?

Reply:
I could not find anywhere a String was used to represent the `value` anywhere in the code unless I am blind and missing something?